src/main/flight/pid.c

```diff
@@ -125,12 +125,12 @@ void resetPidProfile(pidProfile_t *pidProfile)
         .pidSumLimit = PIDSUM_LIMIT,
         .pidSumLimitYaw = PIDSUM_LIMIT_YAW,
         .yaw_lowpass_hz = 0,
-        .dterm_lowpass_hz = 100,    // dual PT1 filtering ON by default
-        .dterm_lowpass2_hz = 200,   // second Dterm LPF ON by default
+        .dterm_lowpass_hz = 150,    // dual PT1 filtering ON by default
```

Comment:
Are these defaults desirable for targets that do not have `USE_DYN_LPF`? If not they should be made conditional.